dnf/cli/commands/install.py

```diff
@@ -155,6 +156,14 @@ def _install_groups(self, grp_specs):
             if self.base.conf.strict:
                 raise
 
+    def _report_alternatives(self, pkg_spec):
+        subj = dnf.subject.Subject(self.alternatives_provide.format(pkg_spec))
+        query = subj.get_best_query(self.base.sack, with_nevra=False,
+                                    with_provides=True, with_filenames=False)
+        if query:
+            msg = _('There are following alternatives for this package: {}')
```

Comment:
What about:
```Available alternatives: {}```
But still not sure.

Reply:
I've enhanced the message a bit, but I'm not sure about shortening it.